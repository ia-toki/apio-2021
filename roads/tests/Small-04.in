c10234cabdfd6eae47773fb6f163e0350fc797e1
181
5 80 816387190
160 138 908601689
145 125 961098468
107 51 182148967
54 17 306520506
25 98 461279198
170 25 867673026
152 103 101308465
64 38 218259664
24 1 190808943
113 180 736130508
62 13 73474652
0 98 287189855
6 121 528919853
22 118 719609142
86 146 325929695
171 170 595582454
51 14 463911988
151 112 38410684
42 125 421120587
93 67 621923151
116 61 324258523
155 106 373240805
176 13 778188812
57 128 272990607
27 70 422254065
76 53 417594945
33 47 934753702
115 116 468724098
3 172 610635267
35 5 164327084
53 81 557108218
73 171 103317473
39 107 260971667
30 81 252003210
7 118 4440704
10 164 89589901
56 77 252908895
60 170 253336331
142 80 932610701
119 1 708144594
95 8 179490721
111 130 702535270
32 131 88913548
12 157 284264359
167 52 756963147
28 130 475871519
150 94 296367942
47 153 11684586
72 98 3676233
114 52 704274498
81 25 433544988
92 20 498066087
87 72 770359744
147 30 546914727
127 179 67560777
177 84 651990863
129 69 735542099
67 12 422200110
104 87 815727906
122 153 66281986
79 47 687146586
154 62 63625730
99 83 193005911
9 20 128112583
15 57 574444842
161 109 582440408
168 147 4532017
88 104 57959121
141 17 1152594
143 130 577526318
45 69 532737492
70 141 502858169
101 108 305213401
31 103 784093070
136 145 324627836
17 72 184664776
117 46 231776387
78 76 642017994
130 24 639516865
120 25 785932076
83 4 201316107
179 69 22623807
43 23 556917204
84 108 891552319
164 1 731648872
1 25 271691028
61 153 20209089
55 12 264898245
123 175 351253514
106 13 855648890
166 31 229402541
4 141 199417387
23 158 41597211
102 57 621529835
103 72 334845976
20 122 908048843
44 72 897254847
126 69 899559788
180 89 79312410
36 84 519084069
52 91 962271062
37 28 323292877
158 20 849318054
97 172 886778405
91 0 142601399
108 119 116348133
69 172 210400203
165 154 938067553
159 24 453874037
14 139 257597912
48 144 883377647
163 82 994055257
109 36 492796187
21 133 465564499
16 70 101033749
137 54 748059063
138 108 747323505
124 87 116479436
11 70 147093526
172 91 409057167
77 1 652149942
85 107 973649348
173 59 177338857
68 118 887709597
132 85 571231973
94 137 320367878
135 180 557629148
71 103 111299979
13 91 370413810
178 112 722580967
26 137 393011162
41 51 878912042
100 24 507817740
133 25 229740165
162 24 229481892
49 69 355982517
18 175 729942234
90 31 340581386
169 0 664053274
134 172 613429267
46 38 484499648
59 105 37714408
82 128 914551666
157 82 934418964
144 153 825498393
58 142 179949341
96 106 639014268
118 0 420651185
146 130 350332795
149 42 57340782
65 1 857750246
110 46 330356557
175 103 468401656
75 28 977083052
40 75 850247283
89 0 391949816
19 128 16549977
148 19 393553908
112 103 962253892
63 83 910701224
105 170 177127779
128 98 506294504
156 68 369350417
29 126 802279551
74 127 449701907
140 151 175989497
153 81 304471406
121 170 60424458
8 91 435188451
38 62 952367774
2 175 178552073
50 77 376621950
131 102 962217221
66 14 607672988
125 119 155613159
80 171 939714555
174 144 544611236
34 61 652863293
139 171 607703468
