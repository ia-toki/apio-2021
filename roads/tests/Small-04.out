064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
84015413152 42747965100 18248850970 5738045583 1706592061 395623300 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
